Orabot.Core.Abstractions/Orabot.Core.Abstractions.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Discord.Net.WebSocket" Version="3.11.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.1" />
```

Comment:
Specifying user secrets as a configuration source feels like an implementation detail rather than a core "contract" detail (which is why you've used it in `Orabot.Core` and not `Orabot.Core.Abstraction`. Did you have a reason to add the package to the `Abstraction` project and not the implementation one?

Reply:
Yes, because I can't use `AddUserSecrets` otherwise which is an extension method from that package.

Reply:
Fixed.